lib/config/options/index.ts

```diff
@@ -2065,6 +2065,8 @@ const options: RenovateOptions[] = [
     description: 'Branch name template.',
     type: 'string',
     default: '{{{branchPrefix}}}{{{additionalBranchPrefix}}}{{{branchTopic}}}',
+    deprecationMsg:
+      'Direct editing of branchName is now deprecated. Please edit branchPrefix, additionalBranchPrefix, or branchTopic instead`',
```

Comment:
```suggestion
      'Directly editing the branchName is deprecated. Please use branchPrefix, additionalBranchPrefix, or branchTopic.`',
```

Can we use `monospace` styling in the deprecation message? If yes, we should add the `monospace` styling to the config option names.

Reply:
I am not sure about this. @viceice wdyt??

Reply:
we can probably use it for the config key words

Reply:
@HonkingGoose I am not sure how to add monospace styling here, can you help